packages/cashier/src/stores/__tests__/account-transfer-store.spec.ts

```diff
@@ -1,13 +1,14 @@
 import AccountTransferStore from '../account-transfer-store';
 import { getCurrencies, validNumber, CFD_PLATFORMS } from '@deriv/shared';
 import { configure } from 'mobx';
-import type { TTransferAccount, TRootStore, TWebSocket } from 'Types';
+import type { TTransferAccount, TWebSocket } from '../../types';
+import { mockStore, TStores } from '@deriv/stores';
```

Comment:
```suggestion
import { mockStore } from '@deriv/stores';
```
@rostislav-deriv Use `TRootStore` instead 🙏🏻